tests/large/test_ide.py

```diff
@@ -80,6 +80,106 @@ def test_default_eclipse_ide_install(self):
         self.wait_and_close()
 
 
+class EclipsePHPIDETests(LargeFrameworkTests):
```

Comment:
don't do this! Just store the command in setUp(), like:
```
self.command_args = '{} de eclipse-php'.format(UMAKE))
```
and replace `self.command('{} ide eclipse-php'.format(UMAKE)) ` with `self.command(self.command_args)) `

self.command () is executing the command (that's why it's a method and not a variable or overriden property)